api/test/nostd/pair_test.cc

```diff
@@ -14,3 +14,65 @@ TEST(PairTest, BasicConstruction)
   EXPECT_EQ(temp_pair.first(), val_1);
   EXPECT_EQ(temp_pair.second(), val_2);
 }
+
+// Tests that the comparison operator returns true when pair values
+//  are the same and returns false when different.
+TEST(PairTest, ComparisonSameValuesReturnTrue){
+
+  int val_1                = 1;
+  int val_2                = 2;
+  pair<int, int> pair_1 = pair<int, int>(val_1, val_2);
+  pair<int, int> pair_2 = pair<int, int>(val_1, val_2);
+
+  EXPECT_EQ(pair_1 == pair_2, true);
```

Comment:
Just FYI: Since pair::operator== is defined I think you can do
  EXPECT_EQ(pair_1, pair_2)
and 
  EXPECT_NE(pair_1, pair_2)

Reply:
When I try, it spits out a number of errors, does that imply that the == operator is not implemented properly?